src/jaegertracing/DynamicLoad.cpp

```diff
@@ -19,31 +19,36 @@
 
 #include <opentracing/dynamic_load.h>
 
+#include "DynamicLoad.h"
 #include "TracerFactory.h"
 #include "jaegertracing/Tracer.h"
 
-int OpenTracingMakeTracerFactory(const char* opentracingVersion,
-                                 const void** errorCategory,
-                                 void** tracerFactory)
+int JaegerTracingMakeTracerFactoryFct(const char* opentracing_version,
```

Comment:
Why?

Reply:
I will revert this change.

Reply:
Thank you. This has been fixed on master. Try this `git rebase master`.

Reply:
change reverted.